src/openms/source/CHEMISTRY/Element.cpp

```diff
@@ -111,10 +114,34 @@ namespace OpenMS
 
   void Element::setIsotopeDistribution(const IsotopeDistribution & distribution)
   {
-    isotopes_ = distribution;
+    isotope_distr_ = distribution;
+    // TODO: remove this fxn
```

Comment:
It looks at a glance like it's only used in `MetaProSIP` and `FeatureFInderAlgorithmPicked`. Should be pretty easy to get rid of